config/routes.rb

```diff
@@ -4,5 +4,7 @@
     resources :applications
   end
 
+  put "/events/:event_id/applications", to: "applications#select"
+
```

Comment:
A shorter and maybe simpler way to add this is having just `put :applications, to: "applications#select"`, but inside the `resources :events` block (everything inside gets prepended with `/events/:event_id/`).

Reply:
Done.